.github/API_REVIEW.md

```diff
@@ -0,0 +1,67 @@
+# API Review
+
+All API changes should go through API review, in addition to our normal code
+review process. We define an API change as
+
+- a change large enough to warrant updating documentation, or
+- a change that increases the maintenance burden of features we offer to our
+  users (i.e., the "API surface area")
+
+For small changes, some or all of these changes can be omitted, but it's best to
+**err on the side of being thorough**. Especially for large changes, you might
+even consider drafting a full-fledged design document.
+
+It's best to go through an API review **before** you start changing the code, so
+that we can offer guidance on how to proceed before getting too caught in the
+weeds.
+
+## Template
+
+Copy/paste this template into a new issue and fill it in to request an API
+review from a maintainer. Remember: depending on the size of your change, it's
+possible to omit some of the sections below.
+
+```md
+#### Summary
+
+> A brief of the new API, including a code sample.
+> Consider where this feature would fit into our documentation,
+> and what the updated documentation would look like.
+
+<!-- TODO -->
+
+#### Motivation
+
+> What is the root of the problem we are trying to solve?
+
+<!-- TODO -->
+
+#### Similar APIs
+
+> Is this new API similar to an existing Stripe API?
+> Are there similar APIs or prior art in other popular projects?
+
+<!-- TODO -->
+
+#### Alternatives
+
+> How else could we implement this feature? Are there any workarounds?
+> Why should we chose this implementation over another?
+
+<!-- TODO -->
+
+#### Scope
+
+> Which interfaces will this apply to? For example, is this specific
+> to one component, or does it affect all Element components?
+> Does this set a precedent for future interfaces we'll add?
+
+<!-- TODO -->
+
+#### Risks
+
+> Are there any security implications (for example, XSS)?
+> Will this feature be easy for users to misuse?
```

Comment:
Nit: This phrasing (as a yes/no question) makes this question a bit trivial to answer -- how about, "What are some ways users can get confused or misuse this feature?"